### PR TITLE
numerical stability for 1 bit quantization

### LINKED ISSUE #73: Numerical Problem Quantizing Model to 1 Bit
**Describe the bug**
Numerical problem occurs when quantize  ResNet56 model on Cifar10 dataset.

**To Reproduce**
1. Quantize  a pretrained ResNet56 model to 1 bit. 
2. Run the uniform/non-unifom learner, quantize and fintune, after a few iterations the accuracy drops to 0.1

**Expected behavior**
The accuracy goes up as fine-tuning proceeds.

**Desktop (please complete the following information):**
 - OS: Centos
 - Browser chrome
 - Version None

**screen shots **
image

**Additional context**
The problem is caused by line 227 in learners/uniform_quantization/utils.py (for the non-uniform quantization the same problem arise) alpha = w_max - w_min. alpha for some weights might some how goes to zeros (or very near 0) during fine-tuning.  Causing an Nan problem. The problem can be solved by adding a small noise term. A PR has been sent.

### PR DESCRIPTION
**Pull request by ylfzr**
Closes #73 

This fix is generally helpful. In case a variable in customized model contains only a few values, chances are that the max value equals to (or very close to) its min value, causing a NaN problem.

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/PocketFlow?pullRequest=74) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/PocketFlow?pullRequest=74) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/PocketFlow?pullRequest=74) it.</sub>
